### PR TITLE
Practicing branching, created ToString().

### PR DESCRIPTION
**Pull request by TheFakeLorLyons**
I added ToString() to species.

### CODE REVIEW

File: Cosminis/DataAccess/EntitesModels/Species.cs [hunk 1]
```diff
@@ -18,5 +18,10 @@ public Species(Models.SpeciesWithEnum Species)
             this.FoodElementIdFk = Species.FoodElementIdFk;
             this.ElementType = Species.ElementToString(Species.ElementType); //This is the only important line that allows us to communicate internally in enum but when we talk to the database it will be string
         }
+
+        public override string ToString()
+        { 
+            return $"SpeciesId: {this.SpeciesId}, SpeciesName: {this.SpeciesName}, Description: {this.Description}, ElementType: {this.ElementType}, BaseStr: {this.BaseStr}, BaseDex: {this.BaseDex}, BaseInt: {this.BaseInt}";
```
Comment (SenLongYu) [inline Cosminis/DataAccess/EntitesModels/Species.cs]: Can you make each item a new line?

**Review by SenLongYu — CHANGES_REQUESTED** [review]

**Review by dgilliam2022 — CHANGES_REQUESTED** [review]
Can you review this very thoroughly?

**Review by SenLongYu — APPROVED** [review]

**Review by dgilliam2022 — APPROVED** [review]

**Comment by SenLongYu** [comment]
LGTM

File: Cosminis/DataAccess/EntitesModels/Species.cs (hunk 1 above)
Comment (TheFakeLorLyons) [reply Cosminis/DataAccess/EntitesModels/Species.cs]: Good call, I changed it.

**Comment by SenLongYu** [comment]
@dgilliam2022 Can you take a look?

**Pull request merged**